homeassistant/components/dreo/entity.py

```diff
@@ -0,0 +1,71 @@
+"""Dreo for device."""
+
+import logging
+
+from hscloud.hscloudexception import (
+    HsCloudAccessDeniedException,
+    HsCloudBusinessException,
+    HsCloudException,
+    HsCloudFlowControlException,
+)
+
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class DreoEntity(Entity):
+    """Representation of a base a coordinated Dreo Entity."""
+
+    def __init__(self, device, config_entry) -> None:
+        """Initialize the Dreo entity."""
+        super().__init__()
+        self._device = device
+        self._config_entry = config_entry
+        self._model = device.get("model")
+        self._device_id = device.get("deviceSn")
+        self._attr_unique_id = device.get("deviceSn")
+        self._attr_name = device.get("deviceName")
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, self._device_id)},
+            manufacturer="Dreo",
+            model=self._model,
+            name=self._attr_name,
+            sw_version=device.get("moduleFirmwareVersion"),
+            hw_version=device.get("mcuFirmwareVersion"),
+        )
+
+    def _try_command(self, mask_error, **kwargs) -> bool:
```

Comment:
```suggestion
    def _try_command(self, mask_error, **kwargs) -> None:
```

You are only raising or returning True, so we can also return None